contribute.html

```diff
@@ -0,0 +1,271 @@
+                                Check out the documentation sources.</p>
+                        </div>
+                    </div>
+                    </a>
+                </div>
+        
+                <div class="col-sm-12 col-md-6">
+                    <a class="card-2" href="https://groups.google.com/forum/#!forum/loopbackjs" target="_blank">
+                    <div class="row">
+                        <div class="col-sm-3">
+                            <img class="icon" src="img/contribute/devchat-icon-1x.png" alt="Contribute icon"/>
+                        </div>
+                        <div class="col-sm-9">
+                            <h5 class="card-title">Participate in the community</h5>
+                            <p class="regular text-gray">The LoopBack community carries on lively discussions in several places. 
+                                Feel free to learn and participate.</p>
+                        </div>
+                    </div>
+                    </a>
+                </div>
+            </div>
+        </div>
+    </section>
+    <hr class="hairline">
+
+    <!-- Governance section -->
+    <section class="even-section">
+        <div class="container">
+            <h3 class="section-heading text-center" style="padding-top: 48px;">Governance</h3>
+            <p class="regular text-center">The LoopBack project governance model follows the sprint and tradition of open source.</p>
+            <p class="large" style="padding-left:190px;padding-right:190px;padding-top:32px;padding-bottom:24px;">
+                LoopBack is an open, inclusive, and tolerant community of people working together to build 
+                a world-class Node framework and tools. We value diversity of individuals and opinions, and seek to operate on
+                consensus whenever possible. We strive to maintain a welcoming, inclusive, and harassment-free environment, 
+                regardless of the form of communication. When consensus is not achievable, we defer to the owners of each 
+                individual module; the powers of the individual owner are kept in check by the ability of the community to 
+                fork and replace dependencies on the individual module and maintainer.</p>
+        </div>
+    </section>
+    <hr class="hairline">
+
+    <div class="action">
+            <div class="container">
+                <div class="row">
+                    <div class="col-md-7">
+                        <h3 style="color:white">Do you have a blog post or other content on LoopBack?</h3>
+                            <p class="p regular" style="color:white">Send us a link and we'll promote your work.</p>
+                    </div>
+                    <div class="col-md-4">
+                        <a class="btn btn-primary" role="button" style="padding-left:60px; padding-right:60px" 
+                        href="https://github.com/strongloop/strongloop.com">Submit Your Content</a>
```

Comment:
this is a private repo. 
@DaveYVR , how does it work today?  If someone has a LB blog post and wants you to promote, what's the proper way to do it?

Reply:
talked to Dave. Before we figured out what we're going to do for this, let's remove it.  We can modify the page once we find a good and sustainable way. Thanks.